libkbfs/bare_root_metadata.go

```diff
@@ -7,61 +7,19 @@ package libkbfs
 import (
 	"errors"
 	"fmt"
-	"strconv"
 
 	"github.com/keybase/client/go/protocol"
 	"github.com/keybase/go-codec/codec"
 )
 
-// MetadataFlags bitfield.
-type MetadataFlags byte
-
-// Possible flags set in the MetadataFlags bitfield.
-const (
-	MetadataFlagRekey MetadataFlags = 1 << iota
-	MetadataFlagWriterMetadataCopied
-	MetadataFlagFinal
-)
-
-// WriterFlags bitfield.
-type WriterFlags byte
-
-// Possible flags set in the WriterFlags bitfield.
-const (
-	MetadataFlagUnmerged WriterFlags = 1 << iota
-)
-
-// MetadataRevision is the type for the revision number.
-// This is currently int64 since that's the type of Avro's long.
-type MetadataRevision int64
-
-// String converts a MetadataRevision to its string form.
-func (mr MetadataRevision) String() string {
-	return strconv.FormatInt(mr.Number(), 10)
-}
-
-// Number casts a MetadataRevision to it's primitive type.
-func (mr MetadataRevision) Number() int64 {
-	return int64(mr)
-}
-
-const (
-	// MetadataRevisionUninitialized indicates that a top-level folder has
-	// not yet been initialized.
-	MetadataRevisionUninitialized = MetadataRevision(0)
-	// MetadataRevisionInitial is always the first revision for an
-	// initialized top-level folder.
-	MetadataRevisionInitial = MetadataRevision(1)
-)
-
-// WriterMetadata stores the metadata for a TLF that is
+// WriterMetadataV2 stores the metadata for a TLF that is
```

Comment:
I was going to do that when introducing the new version to keep merges as painless as possible for now.